internal/pkg/agent/cmd/upgrade.go

```diff
@@ -31,6 +32,14 @@ const (
 	flagPGPBytes       = "pgp"
 	flagPGPBytesPath   = "pgp-path"
 	flagPGPBytesURI    = "pgp-uri"
+	flagForce          = "force"
+)
+
+var (
+	unsupportedUpgradeError   error = errors.New("this agent is Fleet managed and must be upgraded using Fleet")
+	nonRootExecutionError           = errors.New("upgrade command needs to be executed as root for fleet managed agents")
+	skipVerifyNotAllowedError       = errors.New(fmt.Sprintf("\"%s\" flag is not allowed when upgrading a fleet managed agent using the cli", flagSkipVerify))
```

Comment:
Inconsistent casing: should "Fleet" start with a capital "F" or not?

Reply:
I'll resort to using lower case